### PR TITLE
Mcs 233: Interaction Scenes Pair 6: Swap Target Object and Similar Object Either Immediately Visible or Not Visible

### PR DESCRIPTION
**Pull request by chrislong**

### CODE REVIEW

File: scene_generator/pairs.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import objects
 import util
 from geometry import ROOM_DIMENSIONS, MIN_START_DISTANCE_AWAY
+containers = util # only until containers exists, then import containers
```
Comment (ThomasSchellenbergNextCentury) [inline scene_generator/pairs.py:14]: I think this is merged now?

Comment (chrislong) [reply scene_generator/pairs.py:14]: It's in master, but not in this branch. Want me to merge in master and clean this up?

Comment (ThomasSchellenbergNextCentury) [reply scene_generator/pairs.py:14]: Yeah, I just finished my review.  Everything else looks good.  Feel free to cleanup and merge.

**Comment by ThomasSchellenbergNextCentury** [comment]
@chrislong  Whoops, merging the Pair 5 branch causing another conflict here.  Sorry about all that.

**Comment by chrislong** [comment]
I'm not surprised. It's no problem, I'll deconflict

**Pull request merged**